Dockerfile

```diff
@@ -120,15 +120,15 @@ RUN echo ${CACHEBUST}>/dev/null
 RUN CHART_VERSION="v3.13.3"     CHART_FILE=/charts/rke2-canal.yaml             CHART_BOOTSTRAP=true    /charts/build-chart.sh
 RUN CHART_VERSION="1.10.101"    CHART_FILE=/charts/rke2-coredns.yaml           CHART_BOOTSTRAP=true    /charts/build-chart.sh
 RUN CHART_VERSION="1.36.300"    CHART_FILE=/charts/rke2-ingress-nginx.yaml     CHART_BOOTSTRAP=false   /charts/build-chart.sh
-RUN CHART_VERSION="v1.19.7"    CHART_FILE=/charts/rke2-kube-proxy.yaml        CHART_BOOTSTRAP=true    /charts/build-chart.sh
+RUN CHART_VERSION="v1.20.2"     CHART_FILE=/charts/rke2-kube-proxy.yaml        CHART_BOOTSTRAP=true    /charts/build-chart.sh
```

Comment:
Is there a PR in rke2-charts for this yet?

Reply:
<looking at ci> not yet, didn't realize that was a pre-requirement or that updating k8s in rke2 was so complicated. will work on updating the other stuff also.

Reply:
`developer-docs/upgrading_kubernetes.md ` lays it out step-by-step.

Reply:
> `developer-docs/upgrading_kubernetes.md ` lays it out step-by-step.

yes, I am looking at that now, thanks